CHANGELOG.md

```diff
@@ -0,0 +1,43 @@
+## v1.0.0 (2023-10-29)
```

Comment:
This is really neat to see! I'm wondering if or how the GitHub release information will intersect with the information found in this file (for example, https://github.com/cytomining/pycytominer/releases/tag/v0.1). There's a lot of great detail and acknowledgement there which could help promote good dev culture as well as notifying relevant contributors (for ex. mentioning PR or Issue #'s, authors, etc). I can also see how it might not fit into the Commitizen standard and that these are two distinct procedures.

Reply:
I like the idea of keeping both and having them fill separate roles. I think the CHANGELOG.md might be good as a "just the facts" document because it grows in length over time. But still doing the more extensive release notes to acknowledge contributors, promote good dev culture etc. via the Github releases mechanism

Reply:
Added the detailed release notes.

Reply:
Thanks @kenibrewer and @gwaybio ! Maybe the Commitzen content can be used in conjunction with GitHub releases.

Food for thought outside of solely this PR: Commitizen's organized project log might be helpful to highlight within GitHub release content or in the docs. For example, Pylint's use of [Towncrier](https://github.com/twisted/towncrier) (a changelog solution but different implementation than Commitizen) I believe follows to their [Github Releases](https://github.com/pylint-dev/pylint/releases) and [Sphinx docs](https://pylint.readthedocs.io/en/latest/whatsnew/3/3.0/index.html). Solutions like these tend to vary in _where_ and _how_ the information is stored. It could be good to discuss this further in a new issue.